### ISSUE TITLE
Using Softmax Activation Function in ANN_MLP

### ISSUE THREAD

**Issue by AdnanMuhib** [emgucv/emgucv#194]
How can we use Softmax Activation Function in ANN_MLP?
Is there a way to use custom activation function?

**Comment by emgucv** [comment]
I don't think Open CV implements SoftMax in the ANN_MLP class. 

The best bet would be to train the model in tensorflow and import it to Open CV through the DNN module.
